### PR TITLE
Fix IGTV Live

### PR DESCRIPTION
**Pull request by westy92**
`LiveAddPostLiveToIgtvRequest` is now a dead endpoint: https://github.com/harrypython/itsagramlive/issues/40

I added the new endpoints and modified existing ones.

Example usage:

```java
client.sendRequest(new LiveEndBroadcastRequest(broadcastId)).join();
client.sendRequest(new IgtvCreationToolsRequest()).join();
GetThumbnailsResponse thumbs = client.sendRequest(new LiveBroadcastGetPostLiveThumbnailsRequest(broadcastId)).join();
URL thumbUrl = new URL(thumbs.getThumbnails().get(0));
byte[] firstThumbnail = IOUtils.toByteArray(thumbUrl);
RuploadPhotoResponse thumbUpload = client.sendRequest(new RuploadPhotoRequest(
    firstThumbnail,
    "1",
    String.valueOf(System.currentTimeMillis()),
    broadcastId,
    false)
).join();
client.sendRequest(new MediaConfigureToIgtvRequest(
    thumbUpload.getUpload_id(),
    "Title",
    "Caption!,
    true
)).join();
```

### CODE REVIEW

**Review by seth-gravy — APPROVED** [review]

File: src/main/java/com/github/instagram4j/instagram4j/requests/media/ConfigureToIgtvRequest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import com.github.instagram4j.instagram4j.utils.IGUtils;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import okhttp3.Request;
+
+@RequiredArgsConstructor
+@AllArgsConstructor
+public class ConfigureToIgtvRequest extends IGPostRequest<IGResponse> {
```
Comment (jvogit) [inline src/main/java/com/github/instagram4j/instagram4j/requests/media/ConfigureToIgtvRequest.java:16]: Change Class name to `MediaConfigureToIgtvRequest` to match the `path()` endpoint name

**Review by jvogit — CHANGES_REQUESTED** [review]
Sorry for long delay. Looks good to me except for one change to the class name.

File: src/main/java/com/github/instagram4j/instagram4j/requests/media/ConfigureToIgtvRequest.java (hunk 1 above)
Comment (westy92) [reply src/main/java/com/github/instagram4j/instagram4j/requests/media/ConfigureToIgtvRequest.java]: Good catch! Furthermore, it looks like there was already a `MediaConfigureToIgtvRequest` that I have adapted instead. 👍

File: src/main/java/com/github/instagram4j/instagram4j/requests/media/MediaConfigureToIgtvRequest.java [hunk 2]
```diff
@@ -34,6 +35,13 @@ public String path() {
         return MediaConfigureToIgtvResponse.class;
     }
 
+    @Override
+    protected Request.Builder applyHeaders(IGClient client, Request.Builder req) {
```
Comment (jvogit) [inline src/main/java/com/github/instagram4j/instagram4j/requests/media/MediaConfigureToIgtvRequest.java:39]: Seem to be missing an import causing compilation errors

**Review by jvogit — CHANGES_REQUESTED** [review]

**Review by jvogit — APPROVED** [review]

**Comment by jvogit** [comment]
Thanks for contributing!

**Pull request merged**